Data/ExtendedRoadUpgrades.cs

```diff
@@ -23,12 +23,28 @@ internal class ExtendedRoadUpgrades
                     {
                         "en-US",
                         "Quay"
+                    },
+                    {
+                        "zh-Hans",
```

Comment:
Judging by game files I think that locale names should be changed to `zh-HANS` e `zh-HANT` or they won't be loaded.